package.json

```diff
@@ -67,7 +67,9 @@
     "publish:latest": "lerna publish && yarn publish:check",
     "publish:next": "yarn next:publish && yarn next:publish --skip-npm && yarn publish:check",
     "next:publish": "lerna publish --exact --canary=next --npm-tag=next --yes",
-    "publish:check": "node scripts/check-publish.js"
+    "publish:check": "node scripts/check-publish.js",
+    "start:browser": "yarn rebuild:browser && cd examples/browser && yarn start",
```

Comment:
changing folder is not necessary, you should be able to do something like `run start @theia/example-browser` instead.

Reply:
If you do as @akosyakov said, what about the plugins folder, it is defined as "../../plugins" in package.json, so it wouldn't work for user still starting in "example/browser"

Reply:
ah yes you will need to pass `--cwd` to yarn somehow

Reply:
@lmcbout if I update the code to:

`yarn rebuild:browser && run start \"@theia/example-browser\"`

I do see that the `plugins` are unzipped and added? Did you experience a different behavior?